java/test/jmri/jmrix/cmri/CMRISystemConnectionMemoTest.java

```diff
@@ -19,6 +19,25 @@
         Assert.assertNotNull(m);
     }
 
+    @Test
+    public void testNormalizeSystemName() {
+        CMRISystemConnectionMemo m = new CMRISystemConnectionMemo();
+
+        Assert.assertTrue(m.validSystemNameFormat("CS2",'S'));    
```

Comment:
Is this really a valid System Name in C/MRI? Our docs suggest I need two numbers: a node number and an output or input number, or is this referring to something else?